cmd/backfill/main.go

```diff
@@ -267,12 +267,18 @@ func convert(row []string) (*entry, error) {
 
 	// TODO - labelled "Project #" - what does this map to?
 	// usually: "000120"; occaisonally: "001595, 000102, 002014", "FY20-000792,  FY-2021: 000787 and 000924"
-	data.Intake.FundingNumber = null.StringFrom(row[colPrjNum])
-	data.Intake.FundingSource = null.StringFrom(row[colFundSrc])
+
 	if row[colPrjNum] != "" || row[colFundSrc] != "" {
 		data.Intake.ExistingFunding = null.BoolFrom(true)
+		data.Intake.FundingSources = []*models.SystemIntakeFundingSource{
+			{
+				Source:        null.StringFrom(row[colFundSrc]),
+				FundingNumber: null.StringFrom(row[colPrjNum]),
+			},
+		}
```

Comment:
This might be a good time to consider what, if anything, we want to do with this.

For what it's worth, [this doc covers the backfill process](https://github.com/CMSgov/easi-app/blob/edeffae0d6d0c33694fa4828765f3b30064f8059/docs/operations/backfill.md), which was really only used to import Intake Data from outside of EASi. I see no use for this now that EASi is the system of record for this, and almost wonder if it's worth deleting so that we don't have to continue to make changes like this just so the project builds.

Thoughts on this? We could always keep these changes and do it in a separate PR, but I almost view this code more as an archive or "point in time" than something we should have to actively maintain.

Reply:
Yeah that makes sense. If there is no funding source data to import then it seems reasonable to clean it up. I will have to read through the doc and figure out what's all involved. Do you want me to do it in this one today or in another ticket? @ClayBenson94

Reply:
@ClayBenson94 I went ahead and removed funding sources from backfill and updated the commends to reflect that 23 columns are now expected in the spreadsheet